packages/treasury/src/stablecoinMachine.js

```diff
@@ -32,10 +33,26 @@ import {
   makeRatioFromAmounts,
 } from '@agoric/zoe/src/contractSupport/ratio';
 import { AmountMath } from '@agoric/ertp';
+import {
+  buildParamManager,
+  ParamType,
+} from '@agoric/governance/src/param-manager';
+
 import { makeTracer } from './makeTracer';
 import { makeVaultManager } from './vaultManager';
 import { makeLiquidationStrategy } from './liquidateMinimum';
 import { makeMakeCollectFeesInvitation } from './collectRewardFees';
+import {
+  PROTOCOL_FEE_KEY,
+  POOL_FEE_KEY,
+  RECORDING_PERIOD_KEY,
+  CHARGING_PERIOD_KEY,
+  INITIAL_MARGIN_KEY,
+  LIQUIDATION_MARGIN_KEY,
+  INITIAL_PRICE_KEY,
+  INTEREST_RATE_KEY,
+  LOAN_FEE_KEY,
+} from './paramKeys';
 
 const trace = makeTracer('ST');
 
```

Comment:
I think as discussed at today's meeting, we will want anything that is changeable by governance to not be in terms directly as a property. The reason is that users will look at the terms and assume that anything in the terms is immutable. Instead, it might look like:

```
terms: {
  autoswapInstall, 
  loanParams, // the ones that aren't under governance
  // ... other terms
  governedParameters: {
    loanParams: {
      poolFee,
      protocolFee,
    },
  },
}
```